openpype/hosts/nuke/plugins/publish/collect_slate_node.py

```diff
@@ -5,18 +5,22 @@
 class CollectSlate(pyblish.api.InstancePlugin):
     """Check if SLATE node is in scene and connected to rendering tree"""
 
-    order = pyblish.api.CollectorOrder + 0.09
+    order = pyblish.api.CollectorOrder + 0.002
     label = "Collect Slate Node"
     hosts = ["nuke"]
     families = ["render"]
 
     def process(self, instance):
         node = instance.data["transientData"]["node"]
 
-        slate = next((n for n in nuke.allNodes()
-                      if "slate" in n.name().lower()
-                      if not n["disable"].getValue()),
-                     None)
+        slate = next(
+            (
+                n_ for n_ in nuke.allNodes()
+                if "slate" in n_.name().lower()
+                if not n_["disable"].getValue()
+            ),
+            None
+        )
```

Comment:
Admittedly this feels like a very unsafe approach to retrieving a global node. If anyone happens to have any additional node with `slate` (in any form of upper or lowercase) in their scene wouldn't that potentially conflict here?

Not sure what the slate node is intended to be.
Shouldn't we at least filter the `allNodes` to a specific node type classification?